src/python/test/test_dx_bash_helpers.py

```diff
@@ -269,41 +342,77 @@ def test_xattr_parameters(self):
 
             # Build the applet, patching in the bash helpers from the
             # local checkout
-            applet_id = build_app_with_bash_helpers(os.path.join(TEST_APPS, 'xattr_properties'), dxproj.get_id())
+            applet_id = build_app_with_bash_helpers(
+                os.path.join(TEST_APPS, "xattr_properties"), dxproj.get_id()
+            )
 
             # Run the applet
             applet_args = ["-iseq1=A.txt", "-iseq2=B.txt", "-iref=A.txt", "-iref=B.txt"]
-            cmd_args = ['dx', 'run', '--yes', '--watch', applet_id]
+            cmd_args = ["dx", "run", "--yes", "--watch", applet_id]
             cmd_args.extend(applet_args)
             run(cmd_args, env=env)
 
-    @unittest.skipUnless(testutil.TEST_RUN_JOBS, 'skipping test that would run a job')
+    def test_job_identity_token(self):
```

Comment:
I would consider creating a new test suite for tokens...I don't feel that this is the right place...what do you think @kpjensen ?

Reply:
I think it's okay to leave here as similar python standalone test scripts for `dx-upload-all-outputs` exist here.